other_styles/dodgem/object.json

```diff
@@ -0,0 +1,21 @@
+{
+    "id": "auraspecs.music.DODGEM",
+    "authors": [
+        "Karst van Galen Last"
+    ],
+    "version": "1.0",
+    "objectType": "music",
+    "properties": {
+        "tracks": [
+            {
+                "source": "DODGEM.wav",
```

Comment:
Please rename the file to `dodgems.flac`. Rename the directory as well. And also change this line.

Reply:
@Gymnasiast are you sure? Given we don't have flac support yet, I was going to convert to .wav on the CI when generating the parkobj. On the other hand, having flac extension even though it is still a .wav would probably still work.

Reply:
> On the other hand, having flac extension even though it is still a .wav would probably still work.

It is not a .wav any more. Karst replaced the wav with a .flac file, but didn't update the json file initially.

Reply:
You misunderstand me. I want it as .flac to keep the repository size down. The CI can then convert it to .wav and create a .parkobj that works in OpenRCT2. We don't yet support .flac, so it has to be a .wav for this .parkobj to work in NSF.